### PR TITLE
new icon: reactnavigation (original)

### LINKED ISSUE #1803: [ICON REQUEST]: React Navigation
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Popular Routing and navigation for Expo and React Native apps.

### Links and sources

https://reactnavigation.org/

### Additional information

Popularity:
**Similarweb rank: 104,989** [Link](https://www.similarweb.com/website/reactnavigation.org/#overview)
**Github stars: 22.5k** [Link](https://github.com/react-navigation/react-navigation)

### PR DESCRIPTION
**Pull request by Gautam25Raj**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1803**

## Link to prove your SVG is correct and up-to-date.

https://reactnavigation.org/

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -7961,6 +7961,31 @@
...
         ]
     },
+    {
+        "name": "reactnavigation",
+        "altnames": [],
+        "tags": [
+            "routing",
+            "navigation",
+            "react native",
+            "app"
```
Comment (Panquesito7) [inline devicon.json:7971]:
```suggestion
            "app",
            "open-source"
```

**Review by Gautam25Raj — COMMENTED** [review]
Updated devicon.json

**Review by Panquesito7 — APPROVED** [review]
Looks great! Thank you. 🚀

image

**Review by lunatic-fox — APPROVED** [review]
Great work! ✔️
Thank you for this contribution! 🚀

**Pull request merged**